safespace/src/components/Header.js

```diff
@@ -4,8 +4,10 @@ import React from 'react'
 function Header() {
   return (
     <header className="App-header">
-        <h1>SafeSpace</h1>
-        <h2>Get Up Get Going</h2>
+        <h1>Welcome To Your SafeSpace.</h1>
+        <p>Add a hopeful message. See it on your phone later in the day.</p>
+        <p>Get reminded about how special you are.</p>
+        <small>carrier charges apply</small>
```

Comment:
the small tag is an inline element and is meant to be used inside a paragraph tag, note the code syntax examples in the MDN docs here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small